book/src/dev/rfcs/0008-release-planning.md

```diff
@@ -0,0 +1,191 @@
+- Feature Name: `release_planning`
+- Start Date: 2020-09-14
+- Design PR: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra/pull/0000)
+- Zebra Issue: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Release and distribution plans for Zebra.
+
+# Motivation
+[motivation]: #motivation
+
+We need to plan our release and distribution processes for Zebra.  Since these
+processes determine how users get Zebra and place constraints on how we do
+Zebra development, it's important to think through the implications of our
+process choices.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Zebra is developed *library-first*, as a collection of independently useful
+Rust crates composed together to create `zebrad`, the full node implementation.
+This means that our release and distribution processes need to handle
+distribution of the `zebra-*` libraries, as well as of `zebrad`.
+
+The official distribution channels are as follows:
+
+- `zebra-*` libraries are distributed via Cargo, using
+  [crates.io](https://crates.io);
+- `zebrad` is distributed in binary form via Docker images generated in CI
+  (binaries) or in source form via `cargo install`.
+
+The release process is controlled by pushing an appropriate tag to the
+`ZcashFoundation/zebra` git repository.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+(This section should describe the mechanics of the release process in more
+detail, once we have agreement on distribution channels.  Currently, one
+suggestion is described and motivated below).
```

Comment:
Actions from first alpha retrospective:
4. mark non-essential PRs as draft for a few weeks before the release
    (major version / stable releases, like NUs, vs the ones in between)
5. do a dry run of the release tasks before the next release
6. assign release tasks to specific people and schedule them at a specific time